lib/style.css

```diff
@@ -56,6 +56,20 @@ body {
   background-color: <<BACK_PRIMARY>>;
 }
 
+#player.material paper-icon-button.playing::after {
+  content: '';
+  display: block;
+  position: absolute;
+  top: 50%;
+  left: 50%;
+  margin-top: -1.2em;
+  height: 2.4em;
+  width: 2.4em;
```

Comment:
Should have line noted this aswell, this isn't actually editing the theme.  This is adding the pseudo after element.  Not changing googles theme rather adding a bit to make our theme look right.  I'll post screenshots shortly